src/Geometry.jl

```diff
@@ -409,7 +420,7 @@ end
 function Geometry.InterfaceTriangulation(a::DistributedTriangulation,b::DistributedTriangulation)
   trians = map_parts(InterfaceTriangulation,a.trians,b.trians)
   @assert a.model === b.model
```

Comment:
TO-THINK: Should we also restrict that `a.strategy` and `b.strategy` are of the same type? 

I.e.. `@assert typeof(a.strategy)==typeof(b.strategy)` ? 

At present, I am not sure is this is a precondition of the `InterfaceTriangulation` constructor or not. If it was, for obvious reasons it was not being tested before.